tests/testthat/setup.R

```diff
@@ -0,0 +1,3 @@
+options(package_name = "RSQLite")
+
+withr::defer(options(package_name = NULL), teardown_env())
```

Comment:
A bit more context, perhaps:

```suggestion
rlang::local_options(test_package_name = "RSQLite", .frame = teardown_env())
```